core/domain/takeout_service.py

```diff
@@ -76,7 +76,7 @@ def export_data_for_user(user_id):
             feconf.ROLE_ID_MOBILE_LEARNER in user_settings.roles):
         raise NotImplementedError(
             'Takeout for profile users is not yet supported.')
-    exported_data = dict()
+    exported_data = {}
```

Comment:
@Rijuta-s Thanks for raising the PR, do you know which PR raised this issue in the first place and how it got missed?

Reply:
I believe that no PR raised this rather it is a third party library update.(as all the checks pass locally currently. I think running scripts.clean and reinstalling the libraries will produce this locally as well) Not sure though. I did not investigate much as I raised this PR to unblock my PR2.6  #13683.

Reply:
Thanks a lot for raising this PR and we will definitely merge this but knowing the cause will help us avoid such a situation in the future!

> I believe that no PR raised this rather it is a third party library update

But a PR upgrading the third-party lib. would have failed in that case (and not got merged!), right?

Reply:
`use-dict-literal` rule is added to pylint in version 2.10.0 (<a href="https://github.com/PyCQA/pylint/blob/b6923bd7b52c10d09c36ffbedc31abfb9004074c/ChangeLog#L104">PYLINT CHANGELOG</a>).
The version 2.10.0 was released 22 hours ago https://github.com/PyCQA/pylint/pull/4881.

Why are we on the latest release without upgrading pylint version manually?

Reply:
yup. We are installing the latest version due to `
>` than sign. Thanks @ashutoshc8101 !